docs/quickstart/run_model_on_image.md

```diff
@@ -0,0 +1,67 @@
+You can run fine-tuned models on images using Inference.
+
+An Inference server will manage inference. Inference can be run on your local machine, a remote server, or even a Raspberry Pi.
+
+If you need to deploy to the edge, you can use a device like the Jetson Nano. If you need high-performance compute for batch jobs, you can deploy Inference to a server with a GPU.
+
+!!! tip "Tip"
+    Follow our [Run a Fine-Tuned Model on Images](/docs/quickstart/run_model_on_image) guide to learn how to find a model to run.
+
+First, install the `roboflow` Python package:
+
+```
+pip install roboflow
+```
+
+Next, create a new Python file and add the following code:
+
+```python
+from roboflow import Roboflow
+rf = Roboflow(api_key="API_KEY")
+project = rf.workspace().project("MODEL_ENDPOINT")
+model = project.version(VERSION, local="http://localhost:9001").model
```

Comment:
We use localhost here but we haven't taught the user how to start a server yet. Can we add that snippet? Or better yet, can we just show how to run inference on an image the most simple way?

```suggestion
from inference.models.utils import get_roboflow_model
model = get_roboflow_model(...)
result = model.infer(image) #image can be a url string, numpy array, pil image, etc.
```